packages/xo-server/src/xo-mixins/scheduling.mjs

```diff
@@ -92,21 +94,41 @@ export default class Scheduling {
     if (schedule === undefined) {
       throw noSuchObject(id, 'schedule')
     }
-    return schedule.properties
+    return {
+      ...schedule.properties,
+      healthCheckVmsWithTags:
+        schedule.properties.healthCheckVmsWithTags !== undefined
+          ? JSON.parse(schedule.properties.healthCheckVmsWithTags)
+          : undefined,
```

Comment:
Feel free to use [`ifDef`](https://github.com/vatesfr/xen-orchestra/blob/1f9cf458eca2894016c31a2dbc0cfd21f5b5445b/%40xen-orchestra/defined/index.js#L57-L67).

I love this so much, I wonder if I should create a dedicated lib for it.